Riot/ViewController/BugReportViewController.m

```diff
@@ -79,6 +79,7 @@ - (void)viewDidLoad
     _bugReportDescriptionTextView.layer.borderWidth = 1.0f;
     _bugReportDescriptionTextView.text = nil;
     _bugReportDescriptionTextView.delegate = self;
+    _bugReportDescriptionTextView.keyboardAppearance = kRiotKeyboard;
```

Comment:
@daverPL: Hi, thank you for your PR. This is the right fix, but it is not at the right place.
Could you please move this change in the `userInterfaceThemeDidChange` method?